### PR TITLE
Do not register batches which have only non-document operations

### LINKED ISSUE #79: Do not register batches which have only non-document operations
Originally reported in: https://github.com/ckeditor/ckeditor5/issues/781

It appears that batches that operates solely on document fragments are still registered in undo, creating empty undo steps and throwing editor. Such batches should be filtered.

### PR DESCRIPTION
**Pull request by scofalik**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Do not register batches which have only non-document operations. Closes #79. Closes https://github.com/ckeditor/ckeditor5/issues/781.

### CODE REVIEW

File: src/undoengine.js [hunk 1]
```diff
@@ -65,6 +65,13 @@ export default class UndoEngine extends Plugin {
 
 		this.listenTo( this.editor.model, 'applyOperation', ( evt, args ) => {
 			const operation = args[ 0 ];
+
+			// Do not register batch if the operation is not a document operation.
+			// This prevents from creating empty undo steps, where all operations where non-document operations.
```
Comment (Reinmar) [inline src/undoengine.js]: Link to the bug is missing.

Comment (Reinmar) [reply src/undoengine.js]: Unless you consider this completely obvious that undo should not record these changes?

**Comment by Reinmar** [comment]
But I confirm that the fix work.

File: src/undoengine.js (hunk 1 above)
Comment (scofalik) [reply src/undoengine.js]: Well, I consider everything obvious :). I'll add one more line of comment to make it more obvious.

**Comment by scofalik** [comment]
I've completed requested changes.

**Pull request merged**